.github/workflows/coverage-comment.yml

```
@@ -0,0 +1,239 @@
+      uses: erlef/setup-beam@v1
+      with:
+        elixir-version: "1.17.2"
+        otp-version: "27.0"
+
+    - name: Install system dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y bc jq
+
+    - name: Cache dependencies
+      uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)